core/src/main/java/io/confluent/connect/storage/tools/SchemaSourceTask.java

```diff
@@ -98,10 +99,16 @@ public void start(Map<String, String> props) {
       topic = props.get(TOPIC_CONFIG);
       maxNumMsgs = Long.parseLong(props.get(NUM_MSGS_CONFIG));
       multipleSchema = Boolean.parseBoolean(props.get(MULTIPLE_SCHEMA_CONFIG));
-      partitionCount = Integer.parseInt(props.containsKey(PARTITION_COUNT_CONFIG) ?
-                                        props.get(PARTITION_COUNT_CONFIG) : "1");
-      isOutputEnabled = Boolean.parseBoolean(props.containsKey(ENABLE_STDOUT_CONFIG) ?
-                                        props.get(ENABLE_STDOUT_CONFIG) : "true");
+      partitionCount = Integer.parseInt(
+          props.containsKey(PARTITION_COUNT_CONFIG)
```

Comment:
Fair enough. I won't object. Though, if you ask me, when I use ternary operator for a conditional assignment I go for brevity (ideally a one-liner) more than anything else. At least when one of the two terms is simple (null, 1, true, etc)